src/index.ts

```diff
@@ -1,16 +1,4 @@
-import { Web3PluginBase } from "web3";
-
-export class TemplatePlugin extends Web3PluginBase {
-  public pluginNamespace = "template";
-
-  public test(param: string): void {
-    console.log(param);
-  }
-}
-
-// Module Augmentation
-declare module "web3" {
-  interface Web3Context {
-    template: TemplatePlugin;
-  }
-}
+export {ZkSyncPlugin} from './plugin'
```

Comment:
It will be nice to use prettier on all files.
```suggestion
export { ZkSyncPlugin } from './plugin'
```

Reply:
chainsafe eslint config already runs prettier on all the files, but prefers no spaces I think. You can override it with .prettierc file in the root of the repo I think